src/pages/observation-reports/observation-reports.ts

```diff
@@ -119,7 +138,9 @@ export class ObservationReportsPage {
         this.error = "No data found";
         this.utils.stopLoader();
       },
-      { baseUrl: "dhiti", version: this.entityType ? "v2" : "v1" }
+      { baseUrl: "dhiti", version: this.entityType ? "v1" : "v1" }
```

Comment:
Why did we change v2 to v1?

Reply:
here when we are passing the entityType Then its calling the V2 of api ....and in response i was getting Not Found ...so i posted this in group and deepa told me it should be v1 not v2 so i changed